### DISCUSSION TITLE
± (Plus-Minus) Sign and Multiple Answers

### DISCUSSION THREAD

**Discussion by DaAwesomeP** [josdejong/mathjs#2699] [Ideas]
Is there support for a ± (a.k.a. +/- or plus-minus) sign? Take for example the quadratic equation. The plus-minus in the "opposite of `b` plus/minus the square root of `b` square minus 4 times a times c." There could be two answers or two of the same answers (so really one answer). Can (or will) mathjs support this?

Thanks!

**Comment by josdejong** [comment]
Thanks for your suggestion. This isn't supported right now. It could become useful as soon as math.js has an equation solver, as discussed in #38.
